### ISSUE TITLE
Question on Licensing

### ISSUE THREAD

**Issue by jmattlandrum** [inclusive-design/AChecker#108]
Hi,

I don't know if this question is appropriate here, but I thought I would give it a go. Several of my company's customers have been hit with Accessibility lawsuits that are basically of the form "Here is the output of Achecker. Now give us money". Of course, getting a letter like that is distressing, especially if you thought your site was accessible. 

I read over your license. It seems pretty well written. I am not an attorney, but it is doesn't seem like that use of Achecker is a violation of the the license terms, though it seems like the spirit of this software is to assist developers.

Any thought on the matter? I don't know if there is anything to be done, but I thought it wouldn't hurt to reach out.

Thanks,
--Matt

**Comment by gtirloni** [comment]
Hi @jmattlandrum! I don't speak officially for the project but I will give my perspective on this matter.

You're right that AChecker is a service to assist developers in assessing the accessibility of their websites. It's a free service with no warranties but, unfortunately, we seem to be missing a Terms of Service document for it that details that.

I'm also not an attorney but my understanding is that AChecker provides a free service as-is. If people choose to use the AChecker accessibility reports to file lawsuits, it's totally up to them and it's not encouraged by the project. As far as I know, the accessibility guidelines are a requirement in law by many countries but the usage of AChecker itself is not, that's an important distinction.

IMHO, if people are going to file lawsuits they should do so based on the merits of the WCAG guidelines, not on the word of AChecker alone. I'm repeating myself but I think the distinction is important. What matters in most places are the guidelines.

I'm sorry that your company has been hit with lawsuits in this area. I hope that the output of AChecker and other tools (like Google's own [Lighthouse](https://developers.google.com/web/tools/lighthouse) project available in Chrome Dev Tools) can assist in overcoming these obstacles.

The web does need more accessibility testing in general but I personally find it unfortunate that sometimes the push for better accessibility is done through lawsuits but that's just my opinion. I hope AChecker can help in this regard.

**Comment by jmattlandrum** [comment]
Giovanni,

Thanks for that perspective. What I would hope is that we would have a
dialog where we could go through the list to discuss each item (our sites
use a lot of JavaScript and great service from a company called AudioEye),
but unfortunately it seems like a money grab. The attorneys will offer to
settle for a fairly small amount such that individual people will decide
it’s not worth the effort to go to court.  If course, if this strategy
works 10 times, it’s a good deal for the firms.

The authors of Achecker have provided a wonderful free tool to all of us,
and it’s getting taken advantage of. Of course, that’s just my opinion.

I suppose we will grin and bear it for now.

—Matt
